### DISCUSSION TITLE
Asset library like Blender.

### DISCUSSION THREAD

**Discussion by mrbotfield** [godotengine/godot-proposals#9087] [Editor]
I believe an asset library dock.Like Blender would be a great enhancement,to Godot.If you have used aseet library in Blender,you know it would be great.:)

**Comment by Calinou** [comment]
This already exists in the form of the FileSystem dock, which can be put at the bottom of the editor since 4.3.dev3. If you open a folder with 3D scenes in that dock, you can drag-and-drop them to the 3D editor viewport.

**Reply by mrbotfield** [reply]
Thank you Calinou,I never knew that.
